README.md

```diff
@@ -415,19 +415,24 @@ We strive to completely cover all applicable issues with these core labels:
    * __[S-Tolerable](https://github.com/vector-im/element-web/labels/S-Tolerable):__
      Purely cosmetic or low / no impact to users
 
-3. __Priority__ — All issues which are not `T-Other` are assigned a priority:
-   * __[P1](https://github.com/vector-im/element-web/labels/P1):__ Next
-   * __[P2](https://github.com/vector-im/element-web/labels/P2):__ Later
-   * __[P3](https://github.com/vector-im/element-web/labels/P3):__ Eventually
-   * __[P4](https://github.com/vector-im/element-web/labels/P4):__ Interesting —
-     Not yet scheduled, will accept patches
-   * __[P5](https://github.com/vector-im/element-web/labels/P5):__ Dubious —
-     Will not schedule, would consider patches
+3. __Prevalence__ — All issues labeled `T-Defect` are also assigned a prevalence:
+   * __[P-High](https://github.com/vector-im/element-web/labels/P-High):__ Affects most users regularly or impacts most users' first experience (e.g. registration) 
+   * __[P-Medium](https://github.com/vector-im/element-web/labels/P-Medium):__ Affects some users regularly or most users rarely
+   * __[P-Low](https://github.com/vector-im/element-web/labels/P-Low):__ Most users are unlikely to come across this or it is a corner case
 
 4. __Area__ — Most issues are assigned one or several "areas" using one of the
    many `A-` prefixed labels, e.g. `A-Composer` or `A-Spaces`. Each area label
    maps to a group of features or portion of the UI surface in the app.
 
+The Priority label is now deprecated. Defects will now be classed based on their severity and frequency
```

Comment:
Sorry, I don't think I fully understand why to remove the priority labels. While they are used for `T-Defect`, they are also used for `T-Enhancement` and I feel like that can be quite useful for keeping track of what features have some sort of priority. 

While I think the priority labels aren't that useful, I feel like we should have _something_ for tracking the priority of `T-Enhancement`s

Reply:
We are aiming to address enhancement priorities by linking issues to the roadmap and epics (work ongoing).

The problem with enhancements is similar to the issues: there are 100 "will be worked on next" enhancements which is too many for the team to address "next". The implication that all of these are the highest priority is unrealistic and bad practice.

There's is no intention to remove the existing priority tags until those issues are re-assessed and prioritised more appropriately. The aim is to work through P1 first and proceed based on old priorities.

Reply:
We probably need to explain a little bit in this README how prioritisation will concretely work at the same time as deprecating the existing prioritisation labels.

Reply:
An example of what I would want to be able to express is that an issue requesting screen-sharing will be handled before an issue requesting an in-game overlay. I guess I am looking for a way to express our ability to actually add a feature and the number of users looking for it. I am looking for something a bit more general.

Is this what epics would be used for? I feel like trying to order all the VoIP issues for example isn't really possible. Also, as a person who does triage, I am not sure I will have the right to edit epics, so it will be on the team to put everything there. 

There is a chance I am missing what is the exact plan here and in general, I am probably asking dumb questions, sorry :sweat_smile:

Reply:
> We probably need to explain a little bit in this README how prioritisation will concretely work at the same time as deprecating the existing prioritisation labels.

Oops, I totally missed this, sorry. Yeah, I think generally explaining the plan would be great!

Reply:
> An example of what I would want to be able to express is that an issue requesting screen-sharing will be handled before an issue requesting an in-game overlay.

This sounds like a good roadmap candidate 

> I guess I am looking for a way to express our ability to actually add a feature and the number of users looking for it. I am looking for something a bit more general.

This sounds like a perfect candidate for the new tag that I added to indicate how many users it will affect

> Is this what epics would be used for? I feel like trying to order all the VoIP issues for example isn't really possible. Also, as a person who does triage, I am not sure I will have the right to edit epics, so it will be on the team to put everything there.

Epics are used to track work on larger features (e.g. lots of smaller issues that make up a larger feature).
VoIP bugs should be prioritised by how severe they are and how many users they affect. As severity and prevalence are defined, you should be able to use both labels and we will [soon](https://github.com/github/feedback/discussions/4507#discussioncomment-1119635) be able to use label combinations in searches to pull out issues which impact users the most to prioritise those.
Yes, we are aiming for the team is going to be taking a more active role in triage.

> There is a chance I am missing what is the exact plan here and in general, I am probably asking dumb questions, sorry

It's good to get the feedback :)

Reply:
> This sounds like a perfect candidate for the new tag that I added to indicate how many users it will affect

Do you mean the prevalence labels, aren't those for defects, not enhancements?

> VoIP bugs should be prioritised by how severe they are and how many users they affect. As severity and prevalence are defined, you should be able to use both labels and we will [soon](https://github.com/github/feedback/discussions/4507#discussioncomment-1119635) be able to use label combinations in searches to pull out issues which impact users the most to prioritise those.

So should we also give enhancements a severity to track how important they are or should we only use prevalence? 

> Yes, we are aiming for the team is going to be taking a more active role in triage.

What I'd be hoping for is a way for others to also help. As far as I can tell, the team is quite busy so having a way to let not so busy people help would be great

Thanks for the response btw!

Reply:
> > This sounds like a perfect candidate for the new tag that I added to indicate how many users it will affect
> 
> Do you mean the prevalence labels, aren't those for defects, not enhancements?

I think they should be used for enhancements as well, e.g. to separate corner case enhancement requests from those which would benefit most users. I can clarify the language in the PR.

> > VoIP bugs should be prioritised by how severe they are and how many users they affect. As severity and prevalence are defined, you should be able to use both labels and we will [soon](https://github.com/github/feedback/discussions/4507#discussioncomment-1119635) be able to use label combinations in searches to pull out issues which impact users the most to prioritise those.
> 
> So should we also give enhancements a severity to track how important they are or should we only use prevalence?

If you follow the severity label definitions closely, then if an enhancement is a request to fix a workflow issue, then it should really be a defect.

> > Yes, we are aiming for the team is going to be taking a more active role in triage.
> 
> What I'd be hoping for is a way for others to also help. As far as I can tell, the team is quite busy so having a way to let not so busy people help would be great

Yes, of course. I think the roadmap issue is outside the scope of the specific problem around defect triage. With objective labels, we are aiming to make triage easier, more open to all contributors and more effective.

Reply:
> I can clarify the language in the PR.

Yeah, I think that would be great, thanks!

> Yes, of course. I think the roadmap issue is outside the scope of the specific problem around defect triage. With objective labels, we are aiming to make triage easier, more open to all contributors and more effective.

Yeah, sorry for getting a little off-topic :sweat_smile:

Reply:
@SimonBrandner I've updated the wording now :+1: